docs/node_setup.md

```diff
@@ -69,7 +69,7 @@ $ cargo build --release
 
 If you built the binary following the steps above, the path to your binary is:
 ```
-target/release/hydra-dx
+/home/{UNIX_USER_HOME}/HydraDX-node/target/release/hydra-dx
```

Comment:
this is not right. He'd be inside the folder as before. I think the guy suggested more of a change in systemctl script where you should add the /home/{USERNAME}/ path